### ISSUE TITLE
some issues about Oxford5k dataset and train order

### ISSUE THREAD

**Issue by ycwychen** [layumi/2016_person_re-ID#13]
Hello,thank you for your paper and codes.
I have some questions for you to answer.
 1,I have read the paper that mentioned the data set Oxford5k，but I donot konw how  to get it and does the experiment utilize ti ?
2.With  regard to the experiment, in github, after Installation and Dataset is the Test？Do you omit the training step ?
I'm looking forward to your reply ,thank you !

**Comment by layumi** [comment]
Hi @ycwychen 
Sorry for the late response. 
1. Oxford5k is the test set. You can download it from http://www.robots.ox.ac.uk/~vgg/data/oxbuildings/

2. I use the training set in **CNN Image Retrieval Learns from BoW: Unsupervised Fine-Tuning with Hard Examples** (ECCV2016)
You may find it in http://cmp.felk.cvut.cz/cnnimageretrieval/

**Comment by ycwychen** [comment]
Thanks a lot!

**Comment by ycwychen** [comment]
Thank you very much!@layumi
